public/controllers/agent/components/agents-preview.js

```diff
@@ -329,7 +330,7 @@ export const AgentsPreview = compose(
                 addingNewAgent={this.props.tableProps.addingNewAgent}
                 downloadCsv={this.props.tableProps.downloadCsv}
                 clickAction={this.props.tableProps.clickAction}
-                timeService={this.props.tableProps.timeService}
+                formatUIDate={this.props.tableProps.formatUIDate}
```

Comment:
Why not, import the `formatUIDate` and use it directly. In this case, the prop is coming from props. Is that prop, the same function as the service you created?

Reply:
Using directly `formatUIDate` instead of load tableProps as doing before